core/inject.js

```diff
@@ -239,7 +239,7 @@ Blockly.bumpTopObjectsIntoBounds_ = function(workspace) {
   var scrollMetricsInWsCoords = metricsManager.getScrollMetrics(true);
   var topBlocks = workspace.getTopBoundedElements();
   for (var i = 0, block; (block = topBlocks[i]); i++) {
-    Blockly.bumpObjectIntoBounds_(workspace, scrollMetricsInWsCoords, block);
+    goog.module.get('Blockly.bumpObjects').bumpIntoBounds(workspace, scrollMetricsInWsCoords, block);
```

Comment:
This is indeed correct usage of `get`, but I think I'd have opted to do a `goog.module.declareLegacyNamespace` at the top of `bump_objects.js` rather than this `goog.module.get` call, because the former is less likely to be overlooked in future cleanup passes.

Reply:
> because the former is less likely to be overlooked in future cleanup passes.

(Well: I suspect that _I would be less likely to overlook the former in future cleanup passes_, anyway…)